erpnext/stock/doctype/repost_item_valuation/repost_item_valuation.py

```diff
@@ -25,6 +25,25 @@ def validate(self):
 		self.set_status(write=False)
 		self.reset_field_values()
 		self.set_company()
+		self.validate_accounts_freeze()
+
+	def validate_accounts_freeze(self):
+		acc_settings = frappe.db.get_value(
+			"Accounts Settings",
+			"Accounts Settings",
+			["acc_frozen_upto", "frozen_accounts_modifier"],
+			as_dict=1,
+		)
+		if not acc_settings.acc_frozen_upto:
+			return
+		if acc_settings.frozen_accounts_modifier and self.owner in get_users_with_role(
```

Comment:
The doc owner can be forced set to some other owner as well I guess, doesn't seem reliable. Can we check this with the current session user as we do at other places?

Reply:
Updated